docs/src/gettingstarted.md

```diff
@@ -0,0 +1,222 @@
+```@meta
+CurrentModule = BiobakeryUtils
+```
+
+# [Getting Started](@id getting-started)
+
+This is a package for the [julia programming language](http://julialang.org),
+designed for working with the [bioBakery](https://github.com/biobakery/biobakery) family of tools
+for metagenomic analysis of microbial communities.
+Currently, we support [`MetaPhlAn`](https://github.com/biobakery/MetaPhlAn) and [`HUMAnN`](https://github.com/biobakery/HUMAnN).
+
+Read on to learn how to install the package and use it
+to begin using it to uncover insights about your microbial community data!
+If you run into problems, you can [open an issue](https://github.com/BioJulia/BiobakeryUtils.jl/issues/new/choose) on this repository,
+or start a discussion over on [`Microbiome.jl`](https://github.com/BioJulia/Microbiome.jl/discussions/new).
+
+## Installing julia
+
+If this is your first time using julia,
+you'll need to install it by going to the [julia downloads page](https://julialang.org/downloads/)
+and following the instructions for your platform.
+`BiobakeryUtils.jl` should work on any julia version >= 1.6.0.
+
+Alternatively, you can you [`jill.py`](https://github.com/johnnychen94/jill.py),
+which is an easy-to-use python utility for installing julia.
+
+### Launching julia from the terminal
+
+If you download the "app" versions of julia from the downloads page above,
+you may also want to add `julia` to your shell's `$PATH`
+so that you can launch it from your terminal.
+For windows users, you can look [look here](https://julialang.org/downloads/platform/#adding_julia_to_path_on_windows_10)
+for instructions.
+Mac users, [see here](https://julialang.org/downloads/platform/#optional_add_julia_to_path)
+for instructions.
+
+## Making a project
+
+In julia, it's typically a good idea to use ["projects"](https://pkgdocs.julialang.org/v1/environments/)
+to organize your package dependencies
+(this is similar to "environments" that `conda` uses).
```

Comment:
[`conda`](https://github.com/JuliaPy/Conda.jl) since it's the first time it is mentioned